src/components/musette/musette.scss

```diff
@@ -0,0 +1,87 @@
+$button-height: 42px;
+
+.musette-wrapper {
+	position: relative;
+}
+
+.la-musette {
+	column-gap: 20px;
+	cursor: pointer;
+	display: flex;
+	overflow-x: scroll;
+	overflow-y: hidden;
+	position: relative;
+	user-select: none;
+	width: 100%;
+}
+
+.la-musette-offset > :first-child {
+	margin-left: 7.5vw;
+}
+
+.la-musette-offset > :last-child {
+	margin-right: 7.5vw;
+}
+
+.la-musette-no-scrollbar {
+	scrollbar-width: none;
+
+	&::-webkit-scrollbar {
+		display: none;
+	}
+}
+
+//navigation
+.musette-move-left,
+.musette-move-right {
+	background: no-repeat center/20px url("https://www.metmuseum.org/Rodan/dist/svg/banner-link-back.svg");
```

Comment:
I think we shouldn't depend on a file in Rodan here. Probably best to inline it as a data-uri svg. ? What do you think @JosephCurley